### ISSUE TITLE
"Log in" button is too small on fruitsalad (after SR10)

### ISSUE THREAD

**Issue by milescalabresi** [learning-unlimited/ESP-Website#2543]
After Stable Release 10, the "Log In" button's text on the fruitsalad theme is smaller than the "Sign Up" button's (and aligned left, though that may only be noticeable because of the size):
![image](https://user-images.githubusercontent.com/5506191/38468388-f9c20c64-3b12-11e8-8b0c-cbf708e9e4cc.png)

**Comment by milescalabresi** [comment]
Update: this is only happening on some sites. The screenshot above is from BC's site. Here, for example, is Macaulay's. (The text is still off-center, though.)
![image](https://user-images.githubusercontent.com/5506191/38531717-473a1450-3c3f-11e8-909b-aa5841e8e40d.png)

**Comment by benjaminjkraft** [comment]
Did you try shift-refreshing in case it's a caching issue? (Or being masked
on some sites by one!)

**Comment by milescalabresi** [comment]
I did try shift-refreshing multiple times, and I checked for any template overrides that might affect it. It could always be a caching issue!

**Comment by milescalabresi** [comment]
Yes, that looks right. Here's a more stark example, from Vanderbilt:
![image](https://user-images.githubusercontent.com/5506191/38688455-24ae34d4-3e47-11e8-9061-30f22183002f.png)
